src/test/java/SalesforceModules/ForecastingModule.java

```diff
@@ -184,4 +186,89 @@ public static void verifyRoleAssignToUSer() throws InterruptedException {
             catchAssertError(e);
         }
     }
-}
+
+    /* Click on Users Options... */
+
+    public static void clickUsers(String logStep) throws InterruptedException {
+            try {
+                KeywordUtil.isWebElementVisible(ForecastingPage.userOption, "Users Option is visible");
+                KeywordUtil.click(ForecastingPage.userOption, "Click on user options");
+            } catch (Exception e) {
+                catchAssertError(e);
+            }
+        }
+
+        /* Click on New users button...*/
+        public static void clickOnNewUser(String logStep)throws InterruptedException {
+            try {
+                KeywordUtil.switchToIFrame(0, "All Users ~ Salesforce - Developer Edition");
+                KeywordUtil.isWebElementVisible(ForecastingPage.NewUser, "New User is visible");
+                KeywordUtil.waitForVisible(ForecastingPage.NewUser);
+                takeScreenshotAndAttachInReport();
+                KeywordUtil.click(ForecastingPage.NewUser, "Click on New User.");
+                GlobalUtil.getDriver().switchTo().defaultContent();
+            } catch (Exception e) {
+                catchAssertError(e);
+            }
+        }
+
+        /* Enter all New User details.....*/
+        public static void enterNewUserDetails(HashMap<String, String> dataMap, String logStep) throws InterruptedException {
+            try {
+                KeywordUtil.switchToIFrame(0, "New User ~ Salesforce - Developer Edition");
```

Comment:
We are switching to iframe here but where are we switching back to default frame?